lti_consumer/outcomes.py

```diff
@@ -183,7 +183,8 @@ def handle_request(self, request):
             log.debug("[LTI]: %s", error_message)
             return response_xml_template.format(**failure_values)
 
-        real_user = self.xblock.runtime.get_real_user(urllib.parse.unquote(sourced_id.split(':')[-1]))
+        anon_id = unquote(sourced_id.split(':')[-1])
+        real_user = self.xblock.runtime.service(self, 'user').get_user_by_anonymous_id(anon_id)
```

Comment:
this line turns out to have a bug - it should be        
 real_user = self.xblock.runtime.service(self.xblock, 'user').get_user_by_anonymous_id(anon_id)